runners/gradle-plugin/build.gradle.kts

```diff
@@ -83,6 +80,13 @@ pluginBundle {
     }
 }
 
+val javadocJar by tasks.creating(Jar::class) {
+    group = JavaBasePlugin.DOCUMENTATION_GROUP
+    description = "Assembles java doc to jar"
+    archiveClassifier.set("javadoc")
+    from(tasks.javadoc)
+}
```

Comment:
Can we defer this creating?

Reply:
Good point, yes. It's deferred at the moment.

https://github.com/Kotlin/dokka/blob/fe360f7fadc7173be3ae170778a93ebd01962149/build.gradle.kts#L65-L69

I think I used `creating` because the Gradle Publishing and Signing plugins are janky, and don't fully support deferred config. But I added the Javadoc JAR to the integration test publication, so that will make sure it's tested.